### PR TITLE
Turn off learning rate adjustments as experiment

### PR DESCRIPTION
**Pull request by gordonwatts**
This is addressing #20

* Remove all code that adjusts the learning rate
* Remove all the plotting code that worked with the LR
* Remove some old commented out code

This makes significant improvements to the spiky nature of the learning rate.

### CODE REVIEW

**Comment by gordonwatts** [comment]
image
This is an example of what the loss function looks like with everything turned off - much more smooth!
